appveyor.yml

```diff
@@ -76,6 +76,7 @@ install:
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: conda install --yes --quiet conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
+    - cmd: conda install --yes --quiet conda-build=1.20.0
```

Comment:
We should also ping @msarahan who might help us figure out if the latest `conda-build` is broken or if we are doing something wrong.